ai-decision-services/ai-ad-dpp-engine/prepare-config-data-and-app/prepare-config-data-and-app.md

```diff
@@ -0,0 +1,217 @@
+
+Lab 3: Preparation code, config, library and datasets
+===
+
+Fundamentally, you will need the following things ready for setting up the whole infrastructure:
```

Comment:
Consider rewording - Ensure the following resources are setup properly before proceeding to configure infrastructure resources ....
(colon ':' at the end is not needed as the items below are in a bulleted list )